spec/clojure-spec.coffee

```diff
@@ -160,6 +163,12 @@ describe "Clojure grammar", ->
     expect(tokens[1]).toEqual value: "/", scopes: ["source.clojure"]
     expect(tokens[2]).toEqual value: "bar", scopes: ["source.clojure", "meta.symbol.clojure"]
 
+    {tokens} = grammar.tokenizeLine "x"
+    expect(tokens[0]).toEqual value: "x", scopes: ["source.clojure", "meta.symbol.clojure"]
+
+    {tokens} = grammar.tokenizeLine "1foobar"
+    expect(tokens[0]).not.toEqual value: "1foobar", scopes: ["source.clojure", "meta.symbol.clojure"]
```

Comment:
Can this be changed to an explicit `.toEqual`?  That way if the `meta.symbol.clojure` scope changes this test will continue to fail if the behavior regresses.